blueprints/ember-electron/index.js

```diff
@@ -1,20 +1,22 @@
-// Workaround for https://github.com/adopted-ember-addons/ember-electron/issues/320,
-// caused by https://github.com/ember-cli/ember-cli/issues/7431
-Object.keys(require.cache).forEach((filename) => {
-  if (filename.indexOf('fs-extra') !== -1) {
-    delete require.cache[filename];
-  }
-});
-
-const fs = require('fs-extra');
-const path = require('path');
-const { all, denodeify } = require('rsvp');
-
 const Blueprint = require('ember-cli/lib/models/blueprint');
-const efImport = require('electron-forge/dist/api/import').default;
-const { setupForgeEnv, shouldUseYarn } = require('../../lib/utils/yarn-or-npm');
-
-const Logger = require('../../lib/utils/logger');
+const { api } = require('@electron-forge/core');
+const chalk = require('chalk');
+const {
+  electronProjectPath,
+  emberBuildDir,
+  emberTestBuildDir
+} = require('../../lib/utils/build-paths');
+const path = require('path');
+const denodeify = require('denodeify');
+const fs = require('fs');
+const readFile = denodeify(fs.readFile);
```

Comment:
Since this is running on node >= 10, what do you think about using the [native "fs Promises API"](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api) instead of using `denodeify(fs...)`? Also, for cases not covered by fsPromises what do you think about using the [native `util.promisify`](https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_promisify_original)? Seems like a good time to drop another dependency.

Reply:
If you look at `engines` in `package.json`, we still support Node 8, so can't use promises directly. But I didn't realize promisify was rolled into Node in v8, so I'll def switch to that so we can drop the extra dependency!

Reply:
Do we need to keep support for node 8? It will reach end-of-life status in one month, and this is a semver major release.

Reply:
(Just saw that you updated CI to run on 8 so am marking conversation as resolved)